docs/DeveloperGuide.adoc

```diff
@@ -150,11 +166,366 @@ The `Storage` component,
 Classes used by multiple components are in the `seedu.addressbook.commons` package.
 
 == Implementation
-
 This section describes some noteworthy details on how certain features are implemented.
+There are a total of 6 tabs in our application for each feature: Home Tab, Members Tab, Reimbursement Tab,
+Inventory Tab, Cashier Tab, Overview Tab.
+
+=== Home Tab
+This tab will help to show records of individual transactions from miscellaneous spending, revenue from sales and
+cost of buying items to sell.
+
+Each transaction will require an input of its date, description, category, amount
+and member that is accountable for it.
+
+Revenue from each cashier checkout will also be automatically inputted as
+a positive transaction in this tab with the person being the cashier. The inputted transactions that corresponds to
+a spending will be tabulated for each member in the reimbursement tab to keep track of reimbursements.
+
+
+This is the overall Class Diagram of this tab:
+
+.Class Diagram of Home Tab (transaction package)
+
+image::HomeTabClassDiagram.png[]
+
+==== Add Command feature
+The Add command is facilitated by the `VersionedtreasurerPro`. This command requires access to the `Model` of the
+person package which the `AddressBook` implementation is contained in. All fields in the transactions are compulsory
+to be inputted by the user: date, description, category, amount, person full name. The person's name inputted
+has to match a name already existing in the `AddressBook` which is shown in our Members Tab.
+
+The following sequence diagram shows how the add command works:
+
+.Sequence Diagram of Add Command in Home Tab (transaction package)
```

Comment:
Small issue, but perhaps vary the heights of your arrows such that they don't align with each other on either end of the block.